### ISSUE TITLE
Is there any way to get the inner tree?

### ISSUE THREAD

**Issue by pdcook** [ambuc/quadtree#29]
Thanks very much for this nice crate. Here's what I'm trying to do: after inserting a bunch of points, I'd like to query a handle and find all of the regions and subregions it's in, up to the root region. I can see that this data is available in the `inner` field, just by printing the quadtree with the debug `{:?}` formatting.

Is there a way to do this with the current API? If not, I'll look at implementing it myself and submit a pull request and this functionality would be very powerful for my application.

**Comment by ambuc** [comment]
I don't believe that information is currently publicly accessible. PRs implementing a query method exposing these internals would be very welcome. Let me know if you need any help.
